compute_tools/src/http/openapi_spec.yaml

```diff
@@ -10,7 +10,7 @@ paths:
   /status:
     get:
       tags:
-      - Info
+        - Info
```

Comment:
Please, avoid unrelated changes in the PRs

Reply:
Yes, done.